modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/SystemDataSets.tsx

```diff
@@ -84,10 +127,56 @@ const SelectSystemDataSetModalContent = ({
 };
 
 const SystemDataSets = ({
+	importSystemDataSetURL,
+	namespace,
 	systemDataSets,
 }: {
+	importSystemDataSetURL: string;
+	namespace: string;
 	systemDataSets: Array<ISystemDataSet>;
 }) => {
+	const onDeleteClick = ({
```

Comment:
I implemented deletion to help me with testing, although it was strictly not a part of the story. It's basically copy of deletion logic that we already have for custom data sets.